packages/formatters-util/documentation/InputMaskExample.tsx

```diff
@@ -34,14 +36,19 @@ export const InputMaskExample: FC<ExampleComponentProps> = () => {
                     label="Telefonnummer"
                     // Husk å gi plass til mellomrommene som settes inn!
                     maxLength={11}
-                    {...registerWithTelefonnummerMask("telefonnummer", {
+                    {...registerWithTelefonnummerMask("telefonnr", {
                         required: "Du må fylle inn telefonnummeret ditt",
                     })}
-                    errorLabel={form.formState.errors.telefonnummer?.message}
+                    errorLabel={form.formState.errors.telefonnr?.message}
+                />
+                <TextInput label="Fødselsnummer" maxLength={12} {...registerWithFodselsnummerMask("fodselsnr")} />
+                <TextInput label="Kortnummer" maxLength={19} {...registerWithKortnummerMask("kortnr")} />
+                <TextInput label="Kontonummer" maxLength={13} {...registerWithKontonummerMask("kontonr")} />
+                <TextInput
+                    label="Organisasjonsnummer"
+                    maxLength={12}
```

Comment:
```suggestion
                    maxLength={11}
```

Ser ut som det er en off-by-one her. Får `999 999 9999` når jeg makser ut skjemafeltet. Ellers ser det bra ut!